tests/tests_unit/test_api/test_assets.py

```diff
@@ -109,6 +109,14 @@ def test_list(self, mock_assets_response):
         assert "bla" == jsgz_load(mock_assets_response.calls[0].request.body)["filter"]["name"]
         assert mock_assets_response.calls[0].response.json()["items"] == res.dump(camel_case=True)
 
+    def test_list_ag(self, mock_assets_response):
```

Comment:
It's better if test names have more descriptive names, so something like `test_list_with_aggregated_properties_param`. It makes it easier to understand what has gone wrong just by looking at the test suite if the test should ever fail. Doesn't really matter if these method names become overly long

Reply:
fixed